src/components/__tests__/Subthoughts.js

```diff
@@ -140,3 +140,40 @@ it('context view', async () => {
       thoughtsRanked: [{ value: 'b' }, { value: 'ones' }, { value: 'y', rank: 7 }],
     })
 })
+
+// Sort context
+it('normal view', async () => {
```

Comment:
Rather than test the detailed comparison functionality within the Subthoughts component, we should test it lower down, in `util/__tests__/compareThought.js`.

Reply:
Makes sense.

Reply:
Wait till I push my change since I'll need to create at least one test myself.

Reply:
Sure.

Reply:
I acknowledge that I steered you towards the Subthoughts test to begin with, so that was my error.